app/src/main/java/com/example/caregiver/HomeCaregiver.java

```diff
@@ -147,13 +147,36 @@ public void removeCaregiveePopUp(int groupPosition){
         builder.setMessage("Want to remove " + caregivees.get(groupPosition).name + " from your care?")
                 .setPositiveButton("Yes", (dialog, which) ->{
                     String caregiveeId = caregivees.get(groupPosition).id;
+                    setAssignedStatusForCaregivee(caregiveeId);
                     userRef.child("caregivees").child(caregiveeId).removeValue();
                     ref.child(caregiveeId).child("caregivers").child(userId).removeValue();
                     startActivity(new Intent(getContext(), Dashboard.class));
                 }).setNegativeButton("No", null);
         builder.create().show();
     }
 
+    /**
+     * After caregiver deleted caregivee, all tasks assigned to them should be false.
```

Comment:
Function to set assignedStatus to false for the tasks page.

Reply:
This function looks great! Could we rename it to setAssignedStatusToFalse (just so its more intuitive of what it does elsewhere)?